taxi/urls.py

```diff
@@ -1,5 +1,4 @@
 from django.urls import path
-
 from .views import (
```

Comment:
generally, absolute imports are more reliable and should be preferred. From python docs:

>Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path)